cypress/integration/7-crypto/crypto.spec.ts

```diff
@@ -14,73 +14,85 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-/// <reference types="cypress" />
+import { CryptoEvent } from "matrix-js-sdk/src/crypto";
+import { VerificationRequest } from "matrix-js-sdk/src/crypto/verification/request/VerificationRequest";
+import { MatrixClient } from "matrix-js-sdk/src/matrix";
 
-import type { MatrixClient } from "matrix-js-sdk/src/matrix";
 import { SynapseInstance } from "../../plugins/synapsedocker";
+import { UserCredentials } from "../../support/login";
 
-function waitForEncryption(cli: MatrixClient, roomId: string, win: Cypress.AUTWindow): Promise<void> {
-    return new Promise<void>(resolve => {
-        const onEvent = () => {
-            cli.crypto.cryptoStore.getEndToEndRooms(null, (result) => {
-                if (result[roomId]) {
-                    cli.off(win.matrixcs.ClientEvent.Event, onEvent);
-                    resolve();
-                }
-            });
+const waitForVerificationRequest = (cli: MatrixClient): Promise<VerificationRequest> => {
+    return new Promise<VerificationRequest>(resolve => {
+        const onVerificationRequestEvent = (request: VerificationRequest) => {
+            cli.off(CryptoEvent.VerificationRequest, onVerificationRequestEvent);
+            resolve(request);
         };
-        cli.on(win.matrixcs.ClientEvent.Event, onEvent);
+        cli.on(CryptoEvent.VerificationRequest, onVerificationRequestEvent);
     });
-}
+};
 
-describe("Cryptography", () => {
-    beforeEach(() => {
-        cy.startSynapse("default").as('synapse').then(
-            synapse => cy.initTestUser(synapse, "Alice"),
-        );
-    });
+describe("Starting a new DM", () => {
```

Comment:
This test replaces https://github.com/matrix-org/matrix-react-sdk/blob/develop/test/end-to-end-tests/src/scenarios/e2e-encryption.ts . How should we name it? We can test everything in a row: Start DM, encrypted messages and verification.